scripts/install-deps.sh

```diff
@@ -110,18 +110,24 @@ fetch_and_unpack_termbenchpro()
 
 fetch_and_unpack_boxed()
 {
-    local boxed_cpp_version="1.4.2"
+    local boxed_cpp_git_sha="45e7c2e393dd1644f1ab18fb48ba59910c75aad0"
```

Comment:
We also have a cmake variable for minimal boxed-cpp version `BOXED_CPP_MINIMAL_VERSION` in `cmake/ContourThirdParties.cmake` file